commands/docs.go

```diff
@@ -28,17 +27,20 @@ func init() {
 func docsCmdF(cmd *cobra.Command, args []string) error {
 	outDir, _ := cmd.Flags().GetString("directory")
 
-	if fileInfo, err := os.Stat(outDir); os.IsNotExist(err) {
+	fileInfo, err := os.Stat(outDir)
+
+	switch {
```

Comment:
Because of the `gocritic` linter:

```
commands/docs.go:32:2: ifElseChain: rewrite if-else to switch statement (gocritic)
	if os.IsNotExist(err) {
```

Reply:
The complain for `gocritic` is about using `if-else`. That is correct the old code should not have used `if-else`. My proposal would be to just space the code out a bit:
```go
fileInfo, err := os.Stat(outDir)
if err != nil {
	if !os.IsNotExist(err) {
		return err
	}
	
	err = os.Mkdir(outDir, 0755)
	if err != nil {
		return err
	}
}

if !fileInfo.IsDir() {
	return errors.New(fmt.Sprintf("File \"%s\" is not a directory", outDir))
}
```

At the end this is mostly a subjective opinion. Feel free to ignore my comment.